### DISCUSSION TITLE
Function logs limit 4kb seems low

### DISCUSSION THREAD

**Discussion by LasaleFamine** [vercel/vercel#6830] [Help]
Hello,
we are using the Vercel functions for our product but we are seeing our logs (which are redirected to Logflare) cut at 4kb (as per declared limits). 
The problem is that these logs are vital for us and it's difficult to understand how to handle this limit. 
Do you have some best practice to share on how to fix this problem?

**Comment by aleCeres** [comment]
I have the same issue.
